LICENSE.md

```diff
@@ -2,14 +2,12 @@ Wire Network<br />
 Copyright (c) 2024 WIRE NETWORK, INC. and its contributors. All Rights Reserved.<br />
 This software is owned and created by Wire Network, but it builds upon AntelopeIO/leap and EOSIO/eos.
 
-AntelopeIO/abieos<br />
-
-Copyright (c) 2021-2022 EOS Network Foundation (ENF) and its contributors.  All rights reserved. 
+AntelopeIO/wasm-spec-tests<br />
```

Comment:
Why did we change abieos to wasm-spec-tests?

Reply:
I'll defer to @bearcubsvet here as to what content should be here.  We do need to ensure we don't drop copyright references where relevant, but not sure if we've just moved stuff to different places, etc.

Reply:
reviewed and this is mostly just moving things around, not changing/removing things.  Docs will continue to be refined.